packages/webapp-libs/webapp-tenants/src/hooks/useCurrentTenantRole/useCurrentTenantRole.hook.ts

```diff
@@ -1,5 +1,14 @@
 import { useCurrentTenant } from '../../providers';
 
+/**
+ * Hook that retrieves the user's role of the current tenant.
+ *
+ * This hook uses the `useCurrentTenant` hook to get the current tenant's data, and then extracts the role from the
+ * tenant's membership data.
+ *
+ * @returns {string | null} The role of the current tenant if available, or `null` otherwise.
```

Comment:
I think it might be worth checking it against `master` branch where we have updated docusaurus. There were some changes around how `{}` or `[]` are parsed inside comments, so there is a chance that extra escape chars might be needed here, otherwise it might fail to build docs.

Reply:
No errors :)